bitnami/rabbitmq/metadata

```diff
@@ -0,0 +1,3 @@
+| Name      | Description             | Default        |
```

Comment:
This seems like a file created by mistake. Could you remove it?

Reply:
done